crates/swc/tests/vercel/full/next-33088/output/index.js

```diff
@@ -24,10 +24,10 @@ export default function i() {
                         case 1:
                             return [
                                 4,
-                                (r = e.sent()).json()
+                                r.sent().json()
```

Comment:
`r = e` is only true if this line is evaluated

Reply:
r and e doesn't get referenced anywhere, so assignment is removed. You can repplicate this in terser, swc currently only needs more passes to remove unused vars.

Reply:
I see. My bad, thank you!